consensus/pbft/messages.proto

```diff
@@ -36,15 +36,15 @@ package pbft;
 
 message message {
     oneof payload {
-        request request = 1;
+        request_batch request_batch = 1;
         pre_prepare pre_prepare = 2;
         prepare prepare = 3;
         commit commit = 4;
         checkpoint checkpoint = 5;
         view_change view_change = 6;
         new_view new_view = 7;
-        fetch_request fetch_request = 8;
-        request return_request = 9;
+        fetch_request_batch fetch_request_batch = 8;
+        request_batch return_request_batch = 9;
```

Comment:
I did consider this. The digests of the requests comprising a batch need to be communicated somehow. This will allow a replica processing a new-view can to identify the exact requests it's missing from a, err, missing batch. The simplest approach I thought of was to modify the `PQ` message definition so that it carries an array of the digests of the requests included in the batch.

Ultimately I decided it against to keep the invasiveness of the changes to a minimum.

But also, because I think an argument for efficiency and consistency applies here as well. If we were to avoid exchanging batches in this stage, could we argue that we'd be inconsistent, since we treat the `sendBatch/recvRequestBatch` path differently? You can use the "since backups broadcast their requests to the network, it seems like an exceptional scenario when we are missing a full block of requests" argument to question why we have a leader submit a batch of requests in `sendBatch`. (And IIRC  at one point early during this project the thinking was that the leader should indeed only send digests, and the replicas would ask for the missing individual requests. But I think that we decided that just sending the batch right away was a more efficient/practical way of going at it.)